src/app/geolocation.service.ts

```diff
@@ -22,8 +27,13 @@ export class GeolocationService {
     isInvalid: true,
   }
 
+  private jobList: {[key: string]: any}[] = [];
```

Comment:
yeah this is super outdated, sorry I will cut all this.

Reply:
thank u